### PR TITLE
WIP: Add item to fill model

### PR DESCRIPTION
**Pull request by albfan**
Bind fill box and items.

## Summary / How this PR fixes the problem?

First simple connection from items and fill box

## Steps to Test

Add an item, modify rgba on color chooser.

## Screenshots 

fills-box-select-color

## Known Issues / Things To Do

- [x] Modify rgba
- [x] Show only fills box for selected object
- [ ] Support multiple fills box
- [ ] Bind blendingMode

### CODE REVIEW

**Comment by albfan** [comment]
Panel controls work, but now ColorChooser alpha is broken  (do not update the item)

- [ ] fix alpha colorchooser
- [ ] remove all the debug traces
fills-panel

**Comment by Alecaddd** [comment]
You can do this, man!
This is gonna be an incredible milestone 💪

File: src/Layouts/Partials/FillsBoxPanel.vala [hunk 1]
```diff
@@ -58,7 +58,8 @@ public class Akira.Layouts.Partials.FillsBoxPanel : Gtk.Grid {
         add_btn.add (new Gtk.Image.from_icon_name ("list-add-symbolic",
                                                    Gtk.IconSize.SMALL_TOOLBAR));
         add_btn.clicked.connect (() => {
-            fills_list_model.add ();
+            //TODO: What means add another fillitem to an object: gradient???
```
Comment (bilelmoussaoui) [inline src/Layouts/Partials/FillsBoxPanel.vala]: @Alecaddd ^^^

Comment (albfan) [reply src/Layouts/Partials/FillsBoxPanel.vala]: @bilelmoussaoui Alex share some info about that. 

fillings_cut

Implementation (and how it is useful) is clear to me with that

**Comment by albfan** [comment]
Funny thing is that alpha on colorchooser works initially. I think our model is just the item, that could simplify all the implementation. Let's see.

**Comment by albfan** [comment]
Finally! alpha is working from all the entry points (color chooser, text entry).

- [ ] transform opacity needs to be handled in a different way (probably as three new properties: opacity, fill_alpha and stroke_alpha).
- [ ] Add multiple fills is still not implemented.

@bilelmoussaoui @Alecaddd anyway, a review would be appreciated.

**Comment by albfan** [comment]
@Alecaddd @bilelmoussaoui, I'm ok if you think we can left blending mode and multiple fills for another iteration:

akira-fills-and-transform-opacity

**Comment by Alecaddd** [comment]
Indeed, multiple fillings and blending mode can come later in follow up versions.
I'll review this and merge it.

**Pull request merged**